fastapi_routesmanager/__init__.py,sha256=CuWe0kJylW2RpMuuDDvTxbK6C81yFOp8T9tLEX7L6Pk,336
fastapi_routesmanager/exceptions_logger.py,sha256=3yvLu1jSVL-AjcwdZuYVLMdYDITUKgaFpeEx30tW_0c,667
fastapi_routesmanager/headers_logger.py,sha256=csvUgJPyIv8K55CCpJGGERQJksYbwbR81Qeu7Q-pCGY,700
fastapi_routesmanager/routes_manager.py,sha256=y0syGtAFTbICo8qVMiPIBwp3oKzlW-tb8L1Q5LFA2tI,2541
fastapi_routesmanager-0.0.2.dist-info/METADATA,sha256=b01lx1VnG73y57fC-vP-5v2eIQs2TB5_9jJbs38Lwx8,3693
fastapi_routesmanager-0.0.2.dist-info/WHEEL,sha256=yQN5g4mg4AybRjkgi-9yy4iQEFibGQmlz78Pik5Or-A,92
fastapi_routesmanager-0.0.2.dist-info/top_level.txt,sha256=2ARi_yjTivz2981SwPcfTNxengJ3cn5sOTRSlp1xvnY,22
fastapi_routesmanager-0.0.2.dist-info/RECORD,,
